src/components/edititem/index.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { Typography, TextField } from '@material-ui/core'
+
+export class EditItem extends Component {
+  constructor (props) {
+    super(props)
+
+    this.handleKeyPress = this.handleKeyPress.bind(this)
+    this.escFunction = this.escFunction.bind(this);
+  }
+
+  handleKeyPress (e) {
+    if (e.key === 'Enter') {
+      this.props.handleEnter()
+    }
+  }
+
+  escFunction (event) {
```

Comment:
can u please use e, to be consistent with line 13?

Reply:
can you put a todo, to also handle clicks out of the container to be handled as an esc?

Reply:
nevermind this is handled by unblur

Reply:
okay. SO I will mark this as resolved.

Reply:
can u still please make it consistent about e?